about.html

```diff
@@ -200,7 +200,7 @@ <h2 class="logo_dtitle">Digital Youth GTE 2023 Logo</h2>
           <div class="logo_img">
             <a href="#">
               <img
-                src="/SVG/gte-ahmedabad-2023.jpg"
+                src="SVG/gte-ahmedabad-2023.jpg"
                 alt="logo of GTE Global summit 2023"
                 height="200px"
```

Comment:
- [ ] Kindly consider adding a descriptive placeholder to all the `alt` attributes in the about and index page this is good for accessibility

Reply:
thankyou, done!